integration/build.gradle

```diff
@@ -12,17 +12,16 @@ compileJava { options.release = 11 }
 
 repositories {
 	mavenCentral()
-	maven { url 'https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/' }
-	maven { url 'https://broadinstitute.jfrog.io/broadinstitute/libs-release/' }
+	maven { url 'https://broadinstitute.jfrog.io/artifactory/libs-snapshot' }
```

Comment:
1. `/artifactory/` is the new recommended url scheme, it's the same
2. the only things we use from `libs-release` is the sam client, so I only kept it there
3. `libs-snapshot` includes `libs-snapshot-local`, but it's recommended to use a virtual repository (apparently), which is what this is.